### PR TITLE
Feature/milestone 2

### PR DESCRIPTION
**Pull request by jpdf00**
- Prepare the directory structure
- Implement the BooksList component
- Implement the Book component
- Implement the BooksForm component
- Implement the App component
- Initialize the Redux store

### CODE REVIEW

File: src/states/index.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const BOOKS = [
```
Comment (bellom) [inline src/states/index.js]: Oops! according to the instructions on pathwright we need to create the books in `src/index.js`
image

File: src/states/index.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    title: 'Book Test 1',
+    category: 'Action',
+  },
+  {
+    id: Math.floor(Math.random() * 10000),
+    title: 'Book Test 2',
+    category: 'Horror',
+  },
+  {
+    id: Math.floor(Math.random() * 10000),
```
Comment (bellom) [inline src/states/index.js]: ### [Optional]
Kindly be aware that you can use uuidv4 to automatically generate ids. read more 👉 [npm uuid](https://www.npmjs.com/package/uuidv4)

Example:
```
import uuid from 'uuidv4'
id: uuid(): {title: 'Book Test 1', category: 'Action'}
```

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @jpdf00  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best**

**Comment by jpdf00** [comment]
I have followed @bellom's suggestion and used uuidv4 to generate random ids.

**Review by mahmoud717 — APPROVED** [review]
### STATUS: APPROVED ✔️ ✔️ 

**hi @jpdf00**

**Great job with making all of the requirements. the project is approved, you can go ahead and merge this pull request**

**Pull request merged**